File: week4/4.1/Solar System/Solar System/Planet.m
//
//  Planet.m
//  Solar System
//
//  Created by Eric Lanza on 7/21/18.
//  Copyright © 2018 ETLanza. All rights reserved.
//

#import "Planet.h"

@implementation Planet

- (instancetype)initWithName:(NSString *)name diameter:(NSInteger)diameter dayLength:(float)dayLength millionKMsFromSun:(float)millionKMsFromSun
{
    self = [super init];
    if (self) {
        _name = [name copy];
        _imageName = [name lowercaseString];
        _diameter = diameter;
        _dayLength = dayLength;
        _millionKMsFromSun = millionKMsFromSun;
    }
    return self;
}

@end

File: SolarSystem/SolarSystem/Planet.swift
//
//  Planet.swift
//  SolarSystem
//
//  Created by Eric Lanza on 7/3/18.
//  Copyright © 2018 ETLanza. All rights reserved.
//

import UIKit

class Planet {
    var name: String
    var image: UIImage
    var diameter: Double
    var distanceToSun: Double
    var dayLength: Int
    
    init(name: String, image: UIImage, diameter: Double, distanceToSun: Double, dayLength: Int) {
        self.name = name
        self.image = image
        self.diameter = diameter
        self.distanceToSun = distanceToSun
        self.dayLength = dayLength
    }
}
